OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -41,7 +41,13 @@ public HarvesterDockSequence(Actor self, Actor refinery, int dockAngle, bool isD
 			DragOffset = dragOffset;
 			DragLength = dragLength;
 			Harv = self.Trait<Harvester>();
-			StartDrag = self.CenterPosition;
+
+			/*
```

Comment:
It isn't a bug before this PR as this activity is queued when the harvester arrives at the dock. Then in my docking system, the activity is queued while approaching the dock. I can just delete this code but then people might wonder "why don't you pre-compute as before?". Maybe I'd move the comment elsewhere and remove the dead code.